test/test_kinect_frames.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+import tf
+import rospy
+from unittest import TestCase
+
+
+class TestKinectFrames(TestCase):
```

Comment:
Because I don't expect the test cases in this package to grow much, we can use generic name for this py file and the class, something like `test_rgbd_launch.py` and such.